frontend/src/app/features/work-packages/components/wp-table/context-menu-helper/wp-context-menu-helper.service.ts

```diff
@@ -88,6 +90,10 @@ export class WorkPackageContextMenuHelperService {
 
     allowedActions = allowedActions.concat(this.getAllowedParentActions(workPackage));
 
+    // remove some actions on Gantt
+    if (!!workPackage.addRelation && this.wpViewTimeline.isVisible) {
```

Comment:
This checks for permissions to add relations, but we wan't this special menu always when the timeline is visible. I'd also vote for renaming the variable to something more gantt specific.